addons/website_sale/models/sale_order.py

```diff
@@ -172,6 +172,14 @@ def action_preview_sale_order(self):
             action['url'] = f'/@{action["url"]}'
         return action
 
+    def _ensure_sale_order_token(self, ctx):
+        super()._ensure_sale_order_token(ctx)
+        ctx.update({
+            'website_sale_send_recovery_email': True,
+        })
```

Comment:
seems strange to always set to True regardless of the call site :thinking: 
Also, you can probably move the `portal_ensure_token` part in the base method, no need for a hook + override